sdk/core/core-lro/README.md

```diff
@@ -47,300 +53,11 @@ for more information.
-to export the Poller's class directly, and instead only export the already instantiated poller with the `PollerLike` type.
-
-An example of the definition of a client that returns an instantiated poller can be seen below:
-
-```ts
-class Client {
-  public async makePoller: PollerLike<MyOperationState, MyResult> {
-    const poller = new MyPoller({});
-    // It might be preferred to return the poller after the first request is made,
-    // so that some information can be obtained right away.
-    await poller.poll();
-    return poller;
-  }
-}
-
-// No knowledge of the class MyPoller:
-const poller: PollerLike<MyOperationState, MyResult> = myClient.makePoller();
-```
-
-A poller can be created through its constructor, then it can be polled until it's completed.
-At any point in time, the state of the poller can be obtained without delay through the getOperationState method.
-At any point in time, the intermediate forms of the result type can be requested without delay.
-Once the underlying operation is marked as completed, the poller will stop and the final value will be returned.
-
-Here's an example usage of your client's poller:
-
-```ts
-const poller = myClient.makePoller();
-const state: MyOperationState = poller.getOperationState();
-
-// The intermediate result can be obtained at any time.
-const result: MyResult | undefined = poller.getResult();
-
-// The final result can only be obtained after the poller finishes.
-const result: MyResult = await poller.pollUntilDone();
-```
-
-#### Resuming from a previous poller
-
-Pollers by default have a method called `toString` that invokes the operation's `toString` method.
-It can be used to store the state of a poller indefinitely, to then resume by creating another poller at any time in the future.
-
-If an operation's `toString` method is defined as follows:
-
-```ts
-function toString(this: TestOperation): string {
-  return JSON.stringify({
-    state: this.state
-  });
-}
-```
-
-A custom implementation of a poller can deserialize it by receiving this string and converting it back to JSON, like in the following example:
-
-```ts
-export class MyPoller extends Poller<MyOperationState, string> {
-  constructor(
-    baseOperation: string | undefined
-  ) {
-    let state: MyOperationState = {};
-    if (baseOperation) {
-      state = {
-        ...JSON.parse(baseOperation).state,
-        ...state
-      };
-    }
-    const operation = {
-      state,
-      // ...
-    }
-    super(operation);
-  }
-}
-```
+- [The `@azure/core-lro` samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/core-lro/samples).
+- [The `@azure/core-lro` tests](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/core-lro/test).
+- [Azure SDK libraries using `@azure/core-lro`](https://github.com/Azure/azure-sdk-for-js/search?q="from+%40azure%5C%2Fcore-lro"&unscoped_q="from+%40azure%5C%2Fcore-lro").
```

Comment:
I would expect the samples to show the common ways to use core-lro. The link to the search results doesnt seem useful, one can always run search on their own.

Reply:
Sounds good! I can do that.